deploy/runtime/ui/Battery Dispatch Standalone Options BTM.json

```diff
@@ -267,7 +267,7 @@
             "Label": "Battery dispatch choice",
             "Units": " ",
             "Group": "Battery Dispatch",
-            "IndexLabels": "",
+            "IndexLabels": "Peak shaving| Input grid power targets| Input battery power targets| Manual dispatch| Price signal forecast",
```

Comment:
Self consumption appears to be missing in the index labels.

Reply:
This dispatch algorithm doesn't make sense for a standalone system, so I didn't add it to this page. Is there a good place for me to add a comment as to why it's missing here? I'm sure others will have this same confusion...

Reply:
Ah, right! The help documentation is likely the right spot for users. For us coders - does near the list of configurations in ssc make sense https://github.com/NREL/ssc/pull/1049/files#diff-ad752cc0b0fdf4cc68f248c6994ea37977572012346c2829d3efb51d61f0f90bR185 ?